extensions/amp-a4a/0.1/test/testdata/valid_css_at_rules_amp.reserialized.js

```diff
@@ -25,7 +25,7 @@ export const data = {
    ]
 }
 </script>
-</body></html>`,
+</body></html>\0`,
```

Comment:
if this file isnt source code and just for tests, prefix it with test- or suffix with _test so that it isnt globbed by the compiler task